fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -2340,6 +2340,468 @@ countries:
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                chirps-precip-jas:
+                    label: CHIRPS Precip JAS
+                    description: CHIRPS total precipitation for Jul-Sep
+                    path: chad/chirps-precip-jas.zarr
+                    var_names:
+                        value: precipitation
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                chirp-spi-jj:
+                    label: CHIRP SPI Jun-Jul
+                    description: Average SPI in June-July, calculated from CHIRP
+                    path: chad/chirp-spi-jj-jas.zarr
+                    var_names:
+                        value: spi
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                    format: number3
+                chirp-spi-jas:
+                    label: CHIRP SPI JAS
+                    description: Average SPI in July-September, calculated from CHIRP
+                    path: chad/chirp-spi-jas.zarr
+                    var_names:
+                        value: spi
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                    format: number3
+                chirp-rainfall-sep:
+                    label: CHIRP Rain Sep
+                    description: Total rainfall in September according to CHIRP
+                    path: chad/chirp-rainfall-sep-jas.zarr
+                    var_names:
+                        value: prcp
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                    format: number3
+                chirps-dryspell:
+                    label: Dry Spell
+                    description: Longest dry spell, calculated from CHIRPS
+                    path: chad/chirps-dryspell-jas.zarr
+                    var_names:
+                        value: lls
+                        lat: Y
+                        lon: X
+                        time: T
+                    units: days
+                    colormap: precip
+                    lower_is_worse: no
+                    format: number2
+                enso_state: *enso_3mo
+                bad-years-emdat:
+                    label: EM-DAT Bad Years
+                    path: chad/bad-years-emdat.zarr
+                    description: Drought Years collected from EM-DAT
+                    var_names:
+                        value: bad
```

Comment:
If you used `data-conversion-scripts/zarrify-bad-years.py`, then the bad years variable is named `rank`, not `bad`.
That explains the error you reported here: https://github.com/iridl/python-maprooms/pull/514#issuecomment-2738178677